components/organisms/InsightPage/InsightPage.tsx

```diff
@@ -238,6 +239,28 @@ const InsightPage = ({ edit, insight, pageRepos }: InsightPageProps) => {
     setIsModalOpen(false);
   };
 
+  const updateSuggestionsDebounced = useDebounce( async () => {
+    const req = await fetch(`https://api.github.com/search/repositories?q=${encodeURIComponent(`${repoSearchTerm} in:name in:repo:owner/name`)}`, {
```

Comment:
Can we exclude forks and sort by most recently updated?

Reply:
@brandonroberts  Yes, we can sort! Forks might be disabled by default. I didn't find my forked repo by searching.

Also a related link is here.
https://docs.github.com/en/search-github/searching-on-github/searching-in-forks

Reply:
Ok, let's sort repos by most recently updated and we don't need to add anything additional for forks